auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -449,6 +452,7 @@ public String toString() {
              *                   AuthUI#GOOGLE_PROVIDER}. See {@link AuthUI#SUPPORTED_PROVIDERS} for
              *                   the complete list of supported Identity providers
              */
+            @Deprecated
```

Comment:
When deprecating we should always add JavaDoc pointing developers in the right direction.

Reply:
Of course, I haven't added any documentation anywhere. I want to make sure you like the new APIs first. 😄